src/common/options/osd.yaml.in

```diff
@@ -1061,6 +1061,19 @@ options:
   - osd_mclock_max_capacity_iops_ssd
   flags:
   - startup
+- name: osd_mclock_skip_benchmark
```

Comment:
Given that we are not optimizing the mclock_scheduler for filestore, we should either use this or just use wpq for the tests that use https://github.com/ceph/ceph/blob/master/qa/objectstore/filestore-xfs.yaml?

Reply:
> Given that we are not optimizing the mclock_scheduler for filestore, we should either use this or just use wpq for the tests that use https://github.com/ceph/ceph/blob/master/qa/objectstore/filestore-xfs.yaml?

I would lean towards using wpq for filestore tests. I can modify the above yaml to use wpq. In addition, there are a few more places where this can be changed:

1. qa/objectstore_debug/filestore-xfs.yaml
2. qa/suites/rados/objectstore/backends/ceph_objectstore_tool.yaml
3. qa/suites/rados/objectstore/backends/alloc-hint.yaml
4. qa/suites/rados/singleton-nomsgr/all/recovery-unfound-found.yaml

Let me know.

Reply:
Using wpq for filestore OSDs in testing, makes sense to me. 

We'll also need to ensure that filestore OSDs being upgraded to Quincy, continue to use wpq vs bluestore OSDs using mclock scheduler, but we can address this in a separate PR. 

We should find a suitable place in the test suite to exercise `osd_mclock_skip_benchmark`, just to add testing coverage for this option.

Reply:
Couple of changes in the latest update:

1. For the tests that use filestore explicitly (listed above), osd_op_queue is set to WPQ.
2. Added a standalone test to exercise the `osd_mclock_skip_benchmark` option.